appengine/go11x/cloudsql/go.mod

```diff
@@ -1,5 +1,5 @@
 module cloudsql
 
-go 1.11
```

Comment:
The go.mod specifies the minimum version of Go needed to run the code. So, we may want to keep this at 1.11.

Reply:
GAE Go 1.11 is a LTS release, at what point should we consider idiomatic language updates such as pervasive `%w` and raising the minimum version?

Reply:
I'll make the change soon, but I'm out for a few days.

Reply:
Restored 1.11 in go.mod in this PR